### ISSUE TITLE
[Bug] YouTube changes get every IP address used blocked (please sign in)

### ISSUE THREAD

**Issue by mooleshacat** [iv-org/invidious#4978]
**Describe the bug**

I hate calling this a bug it would be nice to have a category to report outages.

The other day I was noticing my instance that uses a VPN was being blocked regardless of which VPN IP I was using. Just a note, I also have inv_sig_helper and po-tokens set up which do not fix the problem.

I closed the ticket after I found a workaround, by opening the browser over the VPN and going to YT. When I got to YT the website was basically blank where the videos normally appear, and there was a message to make a search so the algo could suggest things.

This never was a thing before to my knowledge, and only started happening recently. Once I did a search through a browser, Invidious started to work again. I assume when invidious went to the page to get videos, it was blank, and that caused the issue.

I believe these are attempts to block VPN's using invidious where they will blanket a set of IP addresses and show a blank page to them, asking them to either search a video or sign in. Basically targeting VPN IP's with some sort of "test" or "check" to see if it is a "bot" (or invidious)

Now it doesn't matter which VPN IP I use, it says "please sign in" and even searching a video on the YT website in the web browser will not fix invidious. It will show the videos on the web browser, but invidious still gets the "please sign in" error.

I am not sure if this test/check is being done on ALL IP addresses or just VPN IP addresses. I am too afraid to have my naked IP banned to check if they are doing the same to residential IP's.

TLDR; it appears they have changed their test/block system to block invidious over VPN IP's and they could be testing it there to be used on ALL business/data center IP's which would affect the public instances in the same way.

**Steps to Reproduce**

**Logs**

**Screenshots**

**Additional context**

I am using FreeTube on Windows which uses an Invidious instance ran on a Raspberry Pi 4
I am using a VPN for invidious
I am using inv_sig_helper and po-token for Invidious
